lib/render/ShaderManager.cpp

```diff
@@ -106,7 +106,7 @@ SmartShaderProgram ShaderManager::GetSmartShader(const std::string &name)
 int ShaderManager::LoadResourceByKey(const std::string &key)
 {
 //    printf("Begin Compile %s\n", key.c_str());
-    void *t = GLManager::BeginTimer();
+//    void *t = GLManager::BeginTimer();
```

Comment:
Remove commented code? The same applies to a few other comments.

Reply:
Those are there for debugging in the future